modules/Makefile.kubernetes

```diff
@@ -10,9 +10,10 @@ KUBERNETES_RESOURCE_VALIDATION ?= configmap
 export KUBERNETES_APP
 
 # Kubectl specific settings
+SSH_SOCK ?= ~/tmp/kubectl-$(CLUSTER_BASTION)
```

Comment:
You may also want to use a random value or username for the filename when you move it to prevent collisions